manim/mobject/logo.py

```diff
@@ -75,19 +82,51 @@ def __init__(self, dark_theme=True):
             m_height_over_anim_height * self.M.get_height()
         )
 
+        # Note: "anim" is only shown in the expanded state
+        # and thus not yet added to the submobjects of self.
         self.anim = anim
         self.anim.set_opacity(0)
 
-    def scale(self, scale_factor, **kwargs):
+    def scale(self, scale_factor: float, **kwargs) -> "ManimBanner":
+        """Scale the banner by the specified scale factor.
+
+        Parameters
+        ----------
+        scale_factor
+            The factor used for scaling the banner.
+
+        Returns
+        -------
+        :class:`~.ManimBanner`
```

Comment:
Question: since the function already explicitly specifies its return type, is it also necessary to specify it here?

Reply:
I also thought that this should be the case, however, I couldn't get it to format correctly. I'll investigate Sphinx and type hints a bit more; eventually I hope that we can drop specifying the return type here as well.